src/Movies/Movies.js

```diff
@@ -11,7 +11,7 @@ const Movies = (props) => {
         image={movie.poster_path}
         alt={movie.title}
         key={movie.id}  
-        toggleView={props.toggleView}
+        showChosenMovie={props.showChosenMovie}
       />
```

Comment:
You could save yourself some typing by destructuring the props parameter -

`const Movies = ({movies, showChosenMovie}) => {...}`